tests/components/utility_meter/test_sensor.py

```diff
@@ -18,10 +22,12 @@
     PAUSED,
 )
 from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
     ATTR_ENTITY_ID,
     ATTR_UNIT_OF_MEASUREMENT,
     ENERGY_KILO_WATT_HOUR,
     EVENT_HOMEASSISTANT_START,
+    VOLUME_CUBIC_METERS,
```

Comment:
Cubic meters are quite common in utility_meter use cases... why use it in the tests ? It should be added as a supported unit and use a fake unit in the tests

Reply:
at least I propose not to use a unit that will eventually be valid in the future (breaking the tests)

Reply:
Yes, you're absolutely right. Fixed to use a fake unit.